src/Tizen.NUI.Components/Controls/RecyclerView/Item/DefaultGridItem.cs

```diff
@@ -265,7 +265,10 @@ protected virtual TextLabel CreateLabel(TextLabelStyle textStyle)
         [EditorBrowsable(EditorBrowsableState.Never)]
         protected virtual ImageView CreateImage(ImageViewStyle imageStyle)
         {
-            return new ImageView(imageStyle);
+            return new ImageView(imageStyle)
+            {
+                AccessibilityHighlightable = false,
```

Comment:
this line might not be needed soon, as there is a change which makes ImageView not highlightable by default:
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-toolkit/+/259931/